src/kernel-install/50-depmod.install

```diff
@@ -2,7 +2,14 @@
 # -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 
-[[ $1 == "add" ]] || exit 0
 [[ $2 ]] || exit 1
 
-exec depmod -a "$2"
+if [[ $1 == "add" ]]; then
+    exec depmod -a "$2"
+fi
+
+if [[ $1 == "remove" ]]; then
+    exec rm -f /lib/modules/$2/modules.{alias{,.bin},builtin.bin,dep{,.bin},devname,softdep,symbols{,.bin}}
+fi
```

Comment:
Well, OK, a better way would be for `depmod` to grow an option like `--cleanup`, where it would unlink all the paths it creates. That would be more maintainable than keeping this list here, but of course it'd be a bit before we could depend on that, so you patch would still have to go in for now.